apple/OmnivoreKit/Sources/Views/Article/OmnivoreWebView.swift

```diff
@@ -340,25 +409,35 @@ public enum WebViewDispatchEvent {
   }
 
   private var scriptPropertyLine: String {
-    switch self {
-    case let .handleFontContrastChange(isHighContrast: isHighContrast):
-      return "event.fontContrast = '\(isHighContrast ? "high" : "normal")';"
-    case let .updateLineHeight(height: height):
-      return "event.lineHeight = '\(height)';"
-    case let .updateMaxWidthPercentage(maxWidthPercentage: maxWidthPercentage):
-      return "event.maxWidthPercentage = '\(maxWidthPercentage)';"
-    case let .updateFontSize(size: size):
-      return "event.fontSize = '\(size)';"
-    case let .updateColorMode(isDark: isDark):
-      return "event.isDark = '\(isDark)';"
-    case let .updateFontFamily(family: family):
-      return "event.fontFamily = '\(family)';"
-    case let .saveAnnotation(annotation: annotation):
-      return "event.annotation = '\(annotation)';"
-    case let .speakingSection(anchorIdx: anchorIdx):
-      return "event.anchorIdx = '\(anchorIdx)';"
-    case .annotate, .highlight, .share, .remove, .copyHighlight, .dismissHighlight:
-      return ""
+    get throws {
+      switch self {
+      case let .handleFontContrastChange(isHighContrast: isHighContrast):
+        return "event.fontContrast = '\(isHighContrast ? "high" : "normal")';"
+      case let .updateLineHeight(height: height):
+        return "event.lineHeight = '\(height)';"
+      case let .updateMaxWidthPercentage(maxWidthPercentage: maxWidthPercentage):
+        return "event.maxWidthPercentage = '\(maxWidthPercentage)';"
+      case let .updateTheme(themeName: themeName):
+        return "event.themeName = '\(themeName)';"
+      case let .updateFontSize(size: size):
+        return "event.fontSize = '\(size)';"
+      case let .updateColorMode(isDark: isDark):
+        return "event.isDark = '\(isDark)';"
+      case let .updateFontFamily(family: family):
+        return "event.fontFamily = '\(family)';"
+      case let .saveAnnotation(annotation: annotation):
+        let encoder = JSONEncoder()
```

Comment:
This is the only real change here. Without this, any note with JSON symbols like `"` would fail to save.